stbtrace/stbtrace

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python2
```

Comment:
Would it be possible to use python 3 here? I believe we need to use python 2 for code that depends on BCC, because I don't think we build BCC with python 3 support yet. For everything else though it would be really good to use python 3, since python 2 is going to be EOLd at the end of the year. Maybe we could change the `exec` at the bottom of this script to be an `os.execl()` or similar.

Reply:
Since the python script execs the bcc script it needs to be python2.  I tried it out and added parenthesis to the print statements to get it to run but it couldn't exec the tracing script.